bentoml/_internal/log.py

```diff
@@ -102,13 +108,25 @@ def format(self, record: logging.LogRecord) -> str:
             "tracebacks_show_locals": get_debug_mode(),
             "show_path": get_debug_mode(),  # show log line # in debug mode
         },
+        "build": {
+            "level": "INFO",
+            "filters": ["tracing"],
+            "formatter": "tracing",
+            "()": "rich.logging.RichHandler",
+            "omit_repeated_times": True,
+            "rich_tracebacks": True,
+            "tracebacks_show_locals": get_debug_mode(),
+            "show_path": get_debug_mode(),  # show log line # in debug mode
+        },
     },
     "loggers": {
         "bentoml": {"handlers": ["internal"], "level": "INFO", "propagate": False},
         # circus logger
         "circus": {"handlers": ["circus"], "level": "INFO", "propagate": False},
         "circus.plugins": {"handlers": ["circus"], "level": "INFO", "propagate": False},
         "asyncio": {"handlers": ["internal"], "level": "INFO", "propagate": False},
+        # build & setuptools logger
+        "build": {"handlers": ["build"], "level": "INFO", "propagate": False},
```

Comment:
I'm not sure this is how we want to handle logging going forward, let's discuss with @ssheng before making more changes here.